operator/src/main/helm/coherence-operator/dashboards/alerts-dashboard.json

```diff
@@ -665,10 +667,11 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "count(coherence_cluster_size)",
+          "expr": "count(coherence_partition_assignment_HA_status == 1)",
```

Comment:
we are returning 0 for orphaned and 1 for engandered.  getHAStatus() folded orphaned and endangered into being endangered. harvey wanted me to just return the actual backupstrength number and 0 represented that we already lost data and 1 meant we were at risk.  You could fold into one category by checking for less than or equal to zero. I return -1 if there is no value for mbean attribute HAStatusCode on a node and I believe -1 is filtered out.

Reply:
I was not aware 0 was for orphaned, so i can make <= 1, which will mean any endangered or orphaned.